api-reference/docstubs/api/activitystatistics-get.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Get activityStatistics"
+description: "Read properties and relationships of the activityStatistics object."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Get activityStatistics
+
```

Comment:
In this particular case of activitystatistics-get.md - the partner decided not to expose the GET specific statistic functionality.
GLOBAL:
Taking one step back:
If the CSDL does not/cannot definitively list the supported CRUD operations for an entity, authors would need to continue to pick and include only those CRUD topics that apply. 

Can we have partners include which CRUD operations are supported in the CSDL?
Otherwise we ourselves and authors have to keep in mind that they'll continue to select only those generated CRUD topics that apply.

Reply:
I don't think CSDL allows you to say which methods are supported or not.  Will need to check.  OpenAPI on the other hand IS explicit in what is supported.  If we used that, this would be possible.  BTW - this happens **everywhere**.  Conversely there are lots of navigation properties that are read-only yet we will generate the full CRUD, and authors need to choose which to include.  This is better than not generating them, and having folks incorrectly create topics manually (with incorrect file names, template, titles etc).